.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Note that the rules in CODEOWNERS are NOT additive -
+# for any given file, only the LAST matching rule (and not all matching rules) applies.
+
```

Comment:
Add ppl automatically to review based on what's modified in the PR.

Version 0.1 ofc, to be extended/updated over time